common/app/views/support/package.scala

```diff
@@ -184,7 +184,7 @@ object RenderOtherStatus {
   def apply(result: Result)(implicit request: RequestHeader, context: ApplicationContext) = result.header.status match {
     case 404 => NoCache(NotFound)
     case 410 if request.isJson => Cached(60)(JsonComponent(gonePage, "status" -> "GONE"))
```

Comment:
FWIW I think `JsonComponent` will also be serving a 200 in this case - is that acceptable?